partner_kind/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,16 @@
+This module add kind of partner to partners.
+In the method _get_partner_kind : there si the selection you want to add as partner kind. To mange this générique module, we add 4 selections which can obviously be removed.
+In this selection field you will find :
+- Address : for the company or person linked to a parent
+- User : for company internal person which cannot be concerned by the kind
+- Company : for the other case if it's a company
+- Person : as the field is required, there is a default value
+
+The way to use this module:
+
+In the method _get_partner_kind, add the selection you need as partner kind.
+Once is done,if you want to remove all of the initial recordset, you can also remove the compute field or change it to fit better to your case
+if you only want to remove some of the initial recordset, you have to remove in the compute field the linked conditions
+After, the same thing as to be done in a contrains.
+
+For the tests, as they are linked to this specifiques initials recordsets, thet won't worked if you remove or change this initial recordsets.
```

Comment:
Duplicate of USAGE.srt. 
I'm not sure but I think that the readme needs to describe the module itself, not the potential changes that we can do on it.